filament/backend/src/opengl/OpenGLDriver.cpp

```diff
@@ -1808,9 +1808,9 @@ void OpenGLDriver::updateCubeImage(Handle<HwTexture> th, uint32_t level,
 
     GLTexture* t = handle_cast<GLTexture *>(th);
     if (data.type == PixelDataType::COMPRESSED) {
-        setCompressedTextureData(t, level, 0, 0, 0, 0, 0, 0, std::move(data), &faceOffsets);
+        setCompressedTextureData(t, level, 0, 0, 0, std::max(1u, t->width >> level), std::max(1u, t->height >> level), 0, std::move(data), &faceOffsets);
     } else {
-        setTextureData(t, level, 0, 0, 0, 0, 0, 0, std::move(data), &faceOffsets);
+        setTextureData(t, level, 0, 0, 0, std::max(1u, t->width >> level), std::max(1u, t->height >> level), 0, std::move(data), &faceOffsets);
     }
```

Comment:
The lines are too long, please use this instead:

```
auto width = std::max(1u, t->width >> level);
auto height = std::max(1u, t->height >> level);
if (data.type == PixelDataType::COMPRESSED) {
    setCompressedTextureData(t, level, 0, 0, 0, width, height, 0, std::move(data), &faceOffsets);
} else {
    setTextureData(t, level, 0, 0, 0, width, height, 0, std::move(data), &faceOffsets);
}
```

Reply:
Done in [1eb243ff4694b43b7d29bf3bfca3567435cfd655](https://github.com/google/filament/pull/5018/commits/1eb243ff4694b43b7d29bf3bfca3567435cfd655)